src/psyclone/psyGen.py

```diff
@@ -6777,6 +6883,107 @@ def _ignore_handler(self, *_):
         '''
         return None
 
+    def _create_loop(self, parent, variable_name):
+        '''
+        Create a Loop instance. This is done outside _do_construct_handler
+        because some APIs may want to instantiate a specialized Loop.
+
+        :param parent: The parent of the node.
+        :type parent: :py:class:`psyclone.psyGen.Node`
+        :param str variable_name: Name of the iteration variable
+
+        :return: A new Loop instance
+        :rtype: :py:class:`psyclone.psyGen.Loop`
+
+        '''
+        return Loop(parent=parent, variable_name=variable_name)
+
+    def _process_loopbody(self, loop_body, node):
+        ''' Process the loop body. This is done outside _do_construct_handler
+        becayse some APIs may want to perform specialized actions. By default
```

Comment:
s/becayse/because/ and s/specialized/specialised/

Reply:
fixed